rocAL/rocAL/source/decoders/image/fused_crop_decoder.cpp

```diff
@@ -80,66 +83,99 @@ Decoder::Status FusedCropTJDecoder::decode(unsigned char *input_buffer, size_t i
     };
     actual_decoded_width = max_decoded_width;
     actual_decoded_height = max_decoded_height;
-    // You need get the output of random bbox crop 
+    // You need get the output of random bbox crop
     // check the vector size for bounding box. If its more than zero go for random bbox crop
     // else go to random crop
     unsigned int crop_width, crop_height, x1, y1, x1_diff, crop_width_diff;
     if(_bbox_coord.size() != 0)
     {
         // Random bbox crop returns normalized crop cordinates
-        // hence bringing it back to absolute cordinates 
+        // hence bringing it back to absolute cordinates
```

Comment:
Please check the above lines 46 to 90 where an extra space at end of line has been removed, please match it with previous changes